src/main/java/core/basesyntax/color/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.color;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public Color getRandomColor() {
+        int index = random.nextInt(Color.values().length);
+        Color color = Color.values()[index];
+
+        return color;

```

Comment:
Please don't add redundant empty line. 
Don’t create temporary variables if you can directly return value from the method
Bad example:
```
public User createUser(String name, int age) {
  User user = new User(name, age);
  return user;
}
```
Improved example:
```
public User createUser(String name, int age) {
  return new User(name, age);
}
```